alf/examples/ac_atari.gin

```diff
@@ -1,58 +1,9 @@
-# You need to install following packages
-# pip3 install atari-py opencv-python
-
 # environment config
 import tf_agents.environments.suite_gym
 import alf.environments.wrappers
-# From OpenAI gym wiki:
-# "v0 vs v4: v0 has repeat_action_probability of 0.25
-#  (meaning 25% of the time the previous action will be used instead of the new action),
-#   while v4 has 0 (always follow your issued action)
-# Because we already implements frame_skip in AtariPreprocessing, we should always
-# use 'NoFrameSkip' Atari environments from OpenAI gym
-create_environment.env_name='BreakoutNoFrameskip-v4'
+
```

Comment:
should be renamed as atari.gin? It has nothing to do with ac

Reply:
renamed